scripts/kconfig/kconfigfunctions.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (c) 2018-2019 Linaro
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+# Types we support
+# 'string', 'int', 'hex', 'bool'
+conf_file = os.path.join(os.environ['PROJECT_BINARY_DIR'], "include/generated/generated_dts_board.conf")
+
+dt_defines = {}
+if os.path.isfile(conf_file):
+    with open(conf_file, 'r', encoding='utf-8') as fd:
+        line = fd.readline()
+        while line:
```

Comment:
Could do this:

```python
with open(...) as fd:
    for line in fd:
        ...
```

(And remove the final `readline()` too.)

Reply:
Done